### PR TITLE
Fixed /root/.gnupg permissions in hst-install-ubuntu.sh

### PR DESCRIPTION
**Pull request by gejobj**
Set /root/.gnupg permissions to 700 when the directory is created in hst-install-ubuntu.sh

### CODE REVIEW

File: install/hst-install-ubuntu.sh [hunk 1]
```diff
@@ -662,6 +662,9 @@ fi
 # Define apt conf location
 apt=/etc/apt/sources.list.d
 
+# Create new folder if not all-ready exists
+mkdir /root/.gnupg/ && chmod 700 /root/.gnupg/
```
Comment (danielalexis) [inline install/hst-install-ubuntu.sh]: use `mkdir -p` , it will check if it already exists and wont throw errors if it does

**Review by danielalexis — CHANGES_REQUESTED** [review]
Change the mkdir command

File: install/hst-install-ubuntu.sh (hunk 1 above)
Comment (gejobj) [reply install/hst-install-ubuntu.sh]: I thought -p is only used when you want to create a full directory path like /root/12/34/56
I added -p to mkdir, thanks!

**Review by jaapmarcus — COMMENTED** [review]
Also update hst-install-debian.sh :)

**Comment by gejobj** [comment]
> Also update hst-install-debian.sh :)

Done! :-)
https://github.com/hestiacp/hestiacp/pull/2386

Best regards!

**Pull request merged**